evm/src/lookup.rs

```diff
@@ -64,6 +67,21 @@ pub(crate) fn lookup_helper_columns<F: Field>(
     let num_helper_columns = lookup.num_helper_columns(constraint_degree);
     let mut helper_columns: Vec<PolynomialValues<F>> = Vec::with_capacity(num_helper_columns);
 
+    let looking_cols = lookup
+        .columns
+        .iter()
+        .map(|col| vec![col.clone()])
+        .collect::<Vec<Vec<Column<F>>>>();
+    let lookup_cols = looking_cols
+        .iter()
+        .map(|col| &col[..])
+        .collect::<Vec<&[Column<F>]>>();
```

Comment:
The final type we need for `get_helper_cols` is `&[&[Column<F>]]`, and we only have a `Vec<Column<F>>`. The only way I found to reach the final type was through this intermediary step of creating a `Vec<Vec<_>>`. I could,  change the type in `get_helper_cols` but I'm afraid it would to cloning on the CTL side...